### ISSUE TITLE
Cray libfabric patch required for HPE EX40 systems

### ISSUE THREAD

**Issue by jennfshr** [ovis-hpc/ldms#1472]
Without this patch, configure cannot resolve cray's libfabric libs due to not looking in the `lib64` directory.  I haven't spent time trying to make this flexible and look both in lib and lib64, and I don't see this piece whatsoever in OVIS-4.

That said, some work or guidance to conditionally define it or dynamically resolve the lib dir name in the configure.ac will need to be fashioned as a solution here. 

```patch
diff --git a/configure.ac b/configure.ac
index 13f0a862..ddcaa6e1 100644
--- a/configure.ac
+++ b/configure.ac
@@ -189,10 +189,10 @@ dnl we need libibverbs-devel and librdmacm-devel to support rdma
   if test "\$enable_fabric" = "yes"; then
     AC_ARG_WITH([libfabric],
       AC_HELP_STRING([--with-libfabric], [Specify libfabric location]),
-        [AS_IF([test -d "\$withval/lib"], [LIB_FABRIC="\$withval"],
+        [AS_IF([test -d "\$withval/lib64"], [LIB_FABRIC="\$withval"],
           AC_MSG_ERROR([libfabric not found at \$withval]))],
           AC_MSG_ERROR([must specify --with-libfabric]))
-    LDFLAGS="\$LDFLAGS -L\${LIB_FABRIC}/lib"
+    LDFLAGS="\$LDFLAGS -L\${LIB_FABRIC}/lib64"
     AC_SUBST([LIBFABRIC_INCDIR], [\${LIB_FABRIC}/include])
     saveCPPFLAGS=\$CPPFLAGS
     CPPFLAGS="\$CPPFLAGS -I\${LIB_FABRIC}/include"
```

**Comment by morrone** [comment]
@jennfshr please let us know if the latest b4.4 address that build problem.

**Comment by emdonat** [comment]
Resolved by #1473
